src/utils/urlChecks.ts

```diff
@@ -1,4 +1,4 @@
-export function hasURL(str?: string): boolean {
+export function checkStringForURL(str?: string): boolean {
```

Comment:
i kind of prefer the `hasURL` version tbh, it's more clear that it returns a boolean

this kind of feels like it's more of a validation function

Reply:
ok, @tinaszheng gets `hasURL` and @cmcewen gets `shouldHidePosition`

deal? 😅

Reply:
Deal 🤝